### ISSUE TITLE
CTC Loss Argument "preprocess_collapse_repeated=True"

### ISSUE THREAD

**Issue by TheJaeLal** [hugrubsan/Offline-Handwriting-Recognition-with-TensorFlow#9]
Hey, can you explain why did you use "preprocess_collapse_repeated=True". I looked up tensorflow docs, 
and its different from the default behaviour which has preprocess_collapse_repeated=False.

But setting it to true would make the neural network never learn to output repeated classes.
Won't that be a problem, cuz it may not be able to output words with repeated alphabets like eg. "hood", or "mood" ??

Please correct me if I'm wrong

**Comment by hugrubsan** [comment]
First of all, sorry for the delay.... and yes, you're absolutely right.

At first, the reason why I activate this option is because of the pre-processing of the images, first they are resized and then a filling is added, but the problem you are talking about appears.
I have mates who have been testing models of this type with this option to "false" and the results improve. So indeed, this would be a simple but effective improvement (I speak of this in the conclusions of the project report)

**Comment by TheJaeLal** [comment]
Thanks for clarifying!
